src/neo/SmartContract/Native/ManagementContract.cs

```diff
@@ -44,6 +50,20 @@ internal override void OnPersist(ApplicationEngine engine)
             }
         }
 
+        [ContractMethod(0_01000000, CallFlags.ReadStates)]
+        private long GetMinimumDeploymentFee(StoreView snapshot)
```

Comment:
And maybe it's better to have this value in `Policy` contract.

Reply:
I'm not sure. This is a parameter related to contract deployment, and maybe it should be placed here.

Reply:
I think like @roman-khimov , it's related to the environment. But also it's ok too